### PR TITLE
Belts and walls

### PR DESCRIPTION
**Pull request by KimAndreG**
belts and walls are all done both logic and and implementation

### CODE REVIEW

**Review by OdinBjerga — DISMISSED** [review]
Good job! 😄

File: src/main/java/inf112/skeleton/app/board/Board.java [hunk 1]
```diff
@@ -159,15 +276,150 @@ public boolean checkWin(Robot robot){
...
+            }
+        }
+    }
+
+    /**
+     * check if a robot is on a express belt and if it is. move the robot
+     *
+     * @param robot the robot that should be moved
+     */
+    public void checkExpressBelts(Robot robot){
```
Comment (mrtineide) [inline src/main/java/inf112/skeleton/app/board/Board.java:337]: This never gets called

**Review by mrtineide — DISMISSED** [review]
Ok doki
:shipit:

**Review by mrtineide — APPROVED** [review]
Perfect 
:shipit: 
This is related to #25 i think, but not all done!

**Pull request merged**